PostgreSQL.YAML-tmLanguage

```diff
@@ -84,12 +84,12 @@ repository:
   dollar_quotes:
     patterns:
     - comment: "Assume multiline dollar quote is SQL body!
-        Start if double dollar quote is followed by no dollar quote till line ending.
+        Start if double dollar quote is followed by comment (-- or /**/) or linebreak (\n or \r).
         See match for dollar quotes as string: string.unquoted.dollar.pgsql.
         This could easily support other PL languages like PHP and Ruby -- see PHP heredoc as an example."
       # name: meta.dollar-quote.pgsql
       contentName: meta.dollar-quote.pgsql
-      begin: (\$[\w_]*\$)(?=[^\$]*$)
+      begin: (\$[\w_]*\$)(?:\s*)(?=[-\/\n\r]+)
```

Comment:
Minor...
Maybe we push that whitespace match `(?:\s*)` into the lookahead itself to make it a bit simpler?
Something like this
`(\$[\w_]*\$)(?=\s*[-\/\n\r]+)`

Reply:
Whatever you think is best in this case.
Merging this PR regardless of your decision on this.